acs-apiclient/Resources/layout/layout_titleBarWithExitAndSyncButtons.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
⚠️ This would be more efficient with a frameLayout, or better yet (I think) a linearLayout - please ask me if you need a hand figuring out how to get the same result

Reply:
Done.

Reply:
✅